### PR TITLE
Write `interrogate` action and add missing docstrings

### PR DESCRIPTION
**Pull request by daffidwilde**
To get `interrogate` up to 100%, we were missing a handful of docstrings. These were mainly module strings, but they are important for the API documentation.

### CODE REVIEW

**Comment by daffidwilde** [comment]
Hi @TomJamesW 👋

Would you mind looking through these docstrings when you get the chance? The only reasonably lengthy one is in the `Evaluator` class but I want to make sure they aren't misleading.

File: src/synthgauge/evaluate.py [hunk 1]
```diff
@@ -16,7 +18,24 @@
...
 class Evaluator(object):
-    """ """
+    """
+    The central class in ``synthgauge`` used to hold and evaluate data via
+    metrics or visualisation.
+
+    Parameters
+    ----------
+    real: pandas.DataFrame
+        The real, observed dataset to be evaluated against.
```
Comment (TomJamesW) [inline src/synthgauge/evaluate.py:28]:
```suggestion
        Dataframe containing the real data.
```
Updated for consistency with similar docstrings in the metrics modules. I appreciate this is now less descriptive but should still be clear from context what `real` (and `synth`) data is.

File: src/synthgauge/evaluate.py [hunk 2]
```diff
@@ -16,7 +18,24 @@
...
+    synth: pandas.DataFrame
+        A synthesised version of the real data to be evaluated.
```
Comment (TomJamesW) [inline src/synthgauge/evaluate.py:30]:
```suggestion
        Dataframe containing the synthetic data.
```
As above.

File: src/synthgauge/evaluate.py [hunk 3]
```diff
@@ -16,7 +18,24 @@
 
 
 class Evaluator(object):
-    """ """
+    """
+    The central class in ``synthgauge`` used to hold and evaluate data via
+    metrics or visualisation.
```
Comment (TomJamesW) [inline src/synthgauge/evaluate.py:23]:
```suggestion
    metrics and visualisation.
```
Before it read like you had to choose.

**Review by TomJamesW — COMMENTED** [review]
Hi @daffidwilde,

Looks great. I've made a couple of suggestions, mostly wee things so feel free to take or leave.

T.

File: src/synthgauge/evaluate.py (hunk 3 above)
Comment (daffidwilde) [reply src/synthgauge/evaluate.py:23]: A fair point... but what about all the logicians out there grumbling about the difference between "OR" and "XOR" operators?

**Pull request merged**